<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chaise_producteur_page</class>
 <widget class="QDialog" name="chaise_producteur_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smart cinema</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/new/prefix3/beige.jpg);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>871</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix3/orange.png);</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>producteur</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>831</width>
       <height>451</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix2/marron.jpg);</string>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>ajouter</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>50</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>90</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>130</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>170</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>210</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>50</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>nom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>90</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>prenom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>130</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>id :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>170</y>
         <width>55</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>email : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>210</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>t√©lephone :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>250</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>250</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>adresse :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>340</y>
         <width>151</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix3/orange.png);</string>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>afficher</string>
      </attribute>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>411</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
border-image: whit
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>100</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix3/orange.png);</string>
       </property>
       <property name="text">
        <string>pdf</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>150</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix3/orange.png);</string>
       </property>
       <property name="text">
        <string>imprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>modifier</string>
      </attribute>
      <widget class="QTableView" name="tableView_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>311</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>20</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>110</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>190</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_14">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>150</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>20</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>nom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>70</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>prenom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>id :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>110</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>mail :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>190</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>adresse :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>150</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>telephone :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>280</y>
         <width>141</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix3/orange.png);</string>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>supprimer</string>
      </attribute>
      <widget class="QTableView" name="tableView_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>471</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>id :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_15">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>60</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix3/orange.png);</string>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>chaise</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>831</width>
       <height>451</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix2/marron.jpg);</string>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>ajouter</string>
      </attribute>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>85</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>numero :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>150</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>type :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_16">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>90</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>120</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix3/orange.png);</string>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>150</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
       <item>
        <property name="text">
         <string>NORMAL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VIP</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>afficher</string>
      </attribute>
      <widget class="QTableView" name="tableView_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>561</width>
         <height>371</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>modifier </string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_18">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>71</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>75</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>numero :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>60</y>
         <width>55</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>type :</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>281</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>120</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix3/orange.png);</string>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>60</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
       <item>
        <property name="text">
         <string>NORMAL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VIP</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>supprimer</string>
      </attribute>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>80</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>numero :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_20">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>80</y>
         <width>151</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix3/orange.png);</string>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_6">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>130</y>
         <width>256</width>
         <height>192</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: whit
</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
